src/services/utilServices/__tests__/MailClient.spec.ts

```diff
@@ -10,7 +10,7 @@ import {
 } from "@fixtures/identity"
 import _MailClient from "@services/utilServices/MailClient"
 
-const mockEndpoint = "https://api.postman.gov.sg/v1/transactional/email/send"
+const mockEndpoint = "https://api.postman.gov.sg/v1/transactional/email/"
```

Comment:
doesn't this actually hit the endpoint?

Reply:
There are now multiple endpoints `https://api.postman.gov.sg/v1/transactional/email/send` and `https://api.postman.gov.sg/v1/transactional/email/send/{id}`. I will be appending `send` and `{id}` when appropriate

Reply:
no, what i mean is if this is the actual endpoint and we fail to `mockResolvedValueOnce` (or `mockXxx`) correctly, it will actually hit the network, no?

Reply:
I think axios.post is a mocked function so it should not send any requests out. `mockEndpoint` is used to check if `expect(mockAxios.post).toHaveBeenCalledWith` has been called with the right endpoints: `${mockEndpoint}send` and `${mockEndpoint}1`. I'm not sure if this is what you meant?

Reply:
^oops u right